modules/client-configuration/pages/clients-slemicro.adoc

```diff
@@ -1,14 +1,14 @@
 [[clients-slemicro]]
-= Registering {slea}{nbsp}{micro} Clients
+= Registering {nbsp}{micro} Clients
```

Comment:
Why do we need `{nbsp}`? in front of `{micro}` now? Same for all other changes.

Reply:
@juliogonzalez the nbsp ensures we have the space for styles reasons.

Reply:
Yes, but here without a leading word non-breaking space does not make that much sense.  I think it actually could securely be removed with the leading "{slea}".

Reply:
@jcayouette are you **sure** we won't have now two whitespaces?